build/release.mk

```diff
@@ -85,6 +85,10 @@ endif
 	@for plugin_package in $(PLUGIN_PACKAGES) ; do \
 		cat tmpprepackaged/$$plugin_package.tar.gz | gunzip | tar --wildcards --delete "*windows*" --delete "*linux*" | gzip  > $(DIST_PATH)/prepackaged_plugins/$$plugin_package.tar.gz; \
 		cp tmpprepackaged/$$plugin_package.tar.gz.sig $(DIST_PATH)/prepackaged_plugins; \
+		if [ ! -f $(DIST_PATH)/prepackaged_plugins/$$plugin_package.tar.gz ] || [ ! -f $(DIST_PATH)/prepackaged_plugins/$$plugin_package.tar.gz.sig ]; then \
```

Comment:
Can we go a step further than checking if the file is present, and validate the signature as well? There's a potential the signature is just wrong, and this is an opportune moment to discover that before allowing a successful build. Plus, given we're just doing consecutive http fetches, there's a tiny chance we fetch one while an update is occurring, creating an artificial mismatch.

Reply:
you mean validate through GPG? ahh might be a bit tricky to get the public key, unless it's hosted somewhere or we embed it in the makefile.

Reply:
I like the idea though, would be a great future improvement.

Reply:
I like the idea as well. What we could do is have the public key distributed in `.asc` file and not in memory(as we have now).

Reply:
Alright finally got it working. Added the public key under `build/plugin_publickey.asc` for now.